scripts/minikube-k8sibmc-all-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n.---------------------------------------\n${1}\n'---------------------------------------\n"
+  set -x
+}
+
+set -ex
+
+# This is one of the variables injected by ci-firewall. Its purpose is to allow scripts to handle uniqueness as needed
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+
+case ${1} in
+    k8s)
+        CLUSTER_ON_IBM_CLOUD=true
+        ibmcloud login --apikey $IBMC_developer_APIKEY -a cloud.ibm.com -r eu-de -g "Developer-CI-and-QE"
+        ibmcloud ks cluster config --cluster $IBMC_K8S_CLUSTER_ID
+
+        # Integration tests
+        shout "| Running integration Tests on Kubernetes cluster in IBM Cloud"
+        make test-cmd-project
+        make test-integration-devfile
+
+        shout "Cleaning up some leftover namespaces"
+
+        set +x
+        for i in $(kubectl get namespace -o name); do
+	        if [[ $i == "namespace/${SCRIPT_IDENTITY}"* ]]; then
+	            kubectl delete $i
+            fi
+        done
+        set -x
+
+        odo logout
+        ;;
+    *)
+        echo "Need parameter set to minikube or k8s"
```

Comment:
But we are not doing anything for the case when it's set to `minikube`.

Reply:
you are right, updating